src/components/content-tab-snapshots.tsx

```diff
@@ -187,6 +190,12 @@ function SnapshotRow( {
 		};
 	} else if ( snapshotCreationBlocked ) {
 		tooltipContent = { text: userBlockedMessage };
+	} else if ( isOverLimit ) {
+		tooltipContent = {
+			text: __(
+				'Your site exceeds 2GB in size. Updating this demo site may take considerable amount of time and could exceed the maximum allowed size for a demo site.'
+			),
```

Comment:
Could we use `DEMO_SITE_SIZE_LIMIT_GB` to display the size limit ?

Something like:

```suggestion
			text: sprintf(
				__(
					'Your site exceeds %dGB in size. Updating this demo site may take considerable amount of time and could exceed the maximum allowed size for a demo site.'
				),
				DEMO_SITE_SIZE_LIMIT_GB
			),
```

Same for `overLimitMessage` on creating a demo site.

Reply:
Sounds good, I made changes for this in https://github.com/Automattic/studio/pull/811/commits/29d5ec228a03fbfe9bbea209221b70ce1d94470d

I also added separation between `2 GB` because some languages might use a different order for units and number.